src/sensors.c

```diff
@@ -29,16 +29,21 @@ void init_sensors(void)
   uint16_t acc1G;
   float gyro_scale;
   mpu6050_init(true, &acc1G, &gyro_scale);
-  _accel_scale = 9807 / acc1G; // convert to mm/s^2
-  _gyro_scale = (int32_t) (1e6 * gyro_scale); // convert to urad/s
+
+  _accel_scale = 2394; // convert to um/s^2
```

Comment:
So, the problem is that acc1G and gyro_scale in his code are not dynamic, and they are wrong.  I'm hopefully pull requesting some changes to his code so that they will be right.